python/ray/data/impl/plan.py

```diff
@@ -275,13 +285,45 @@ def stats(self) -> DatasetStats:
         self.execute()
         return self._snapshot_stats
 
+    def _should_clear_input_blocks(
+        self,
+        blocks: BlockList,
+        stage_idx: int,
+        clear_input_blocks: bool,
```

Comment:
Does this make sense (since I didn't see response or change)? Basically my thinking is a function like "bool do_something(bool dont_do_if_true, others)" should remove "dont_do_if_true" out of the parameters.

Reply:
Yep I gave a 👍 and made the change!